src/aerie_cli/aerie_client.py

```diff
@@ -1488,49 +1488,53 @@ def get_scheduling_goals_by_specification(self, spec_id):
             scheduling_specification_goals(where: {
                 specification_id:{_eq:$spec}
             }){
-                goal{
+                goal_metadata {
                     id
-                    model_id
                     name
                     description
-                    author
-                    last_modified_by
-                    created_date
-                    modified_date
+                    public
+                    owner
+                    updated_by
+                    created_at
+                    updated_at
                 }
             }
         }
         """
         resp = self.aerie_host.post_to_graphql(list_all_goals_by_spec_query, spec=spec_id)
 
         return resp
-
+    
     def upload_scheduling_goal(self, model_id, name, definition):
         obj = dict()
-        obj["name"] = name
-        obj["model_id"] = model_id
         obj["definition"] = definition
-
         return self.upload_scheduling_goals([obj])
```

Comment:
Does this need to build the `update_scheduling_goals` object structure?

Reply:
Tbh I think we can remove this method entirely, it is not being used anywhere.

Reply:
Even better!

Reply:
Updated in 1d42f2df28796f48635a9953580ac9829dd104f0 to remove `upload_scheduling_goal`, as `upload_scheduling_goals` can just be used directly if needed.